app/assets/src/components/utils/format.js

```diff
@@ -2,3 +2,21 @@ export const formatPercent = number => `${(100 * number).toFixed(1)}%`;
 
 export const limitToRange = (number, min, max) =>
   Math.min(max, Math.max(min, number));
+
+// Converts file sizes to readable format.
+// Example: 1234 to 1.2 kb
+export const formatFileSize = bytes => {
```

Comment:
I'm a little hesitant. It feels a bit premature, and I could see other use cases wanting a different set of units. For example, "mm, cm". Would prefer to generalize when other use cases come up.

Reply:
My requested changes are simple and do not affect this implementation at all, so it would worth doing now, to incentivize people to use it... think if the change can incentivize future developers to use it.
To use this, they might have to change Upload code to adapt which my serve as deterrent. 

`mm` and `cm` is the opposite case (< 1) and so, that could be an extra generalization. Makes sense here, but since it is a bit more work my be worth to do later.

Anyway, will leave it up to you...

Reply:
Okay, factored it out into a separate function.